src/components/auth/SignInForm.tsx

```diff
@@ -7,8 +7,13 @@ import { signInValidationRules } from 'validation/auth'
 import ErrorText from '../shared/Text/ErrorText'
 import { TextInputField } from '../shared/FormFields'
 
+type SignInFormData = {
+  email: string
+  password: string
+}
```

Comment:
We should use interfaces instead of types here. 
There are similarities, but there are few advantages why should we use an interface when we are defining components props. 
https://blog.logrocket.com/types-vs-interfaces-in-typescript/

```suggestion
interface SignInFormData {
  email: string
  password: string
}
```